build/build-sdk-images/rust/build-sdk-test.sh

```diff
@@ -15,6 +15,11 @@
 # limitations under the License.
 
 set -ex
+
+# We need to copy the proto files to the SDK otherwise the build will fail if
+# we only run this script
+"$(dirname "$0")/gen.sh"
```

Comment:
Apologies - we actually don't need this, since as a manual step of https://github.com/googleforgames/agones/tree/main/build#make-gen-sdk-grpc will be run whenever a proto changes, so we don't need to run it here 👍🏻 

So you can remove these changes.

Reply:
Oh the reason I did this is because of the build failure https://github.com/googleforgames/agones/pull/2112#issuecomment-863871807, which was surprising to me, but since we're going to vendor them anyways it doesn't matter.